Dockerfile

```diff
@@ -96,13 +96,15 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
     iproute2 \
     kmod \
     libasan3 \
+    libICE6 libX11-6 libX11-data libX11-xcb1 libXau6 libXdamage1 libXext6 libXfixes3 libXft2 libXmu6 libXmuu1 libXrender1 libXss1 libXt6 libXxf86vm1 \
```

Comment:
I've gone with something more in the middle, mixing libX* and libx* so it is easier to see them together. (there wont be many other cases where case matters for the order)